Sources/Public/CalendarView.swift

```diff
@@ -1259,6 +1259,13 @@ extension CalendarView {
     let accessibilityScrollText = targetMonthView.accessibilityLabel
     UIAccessibility.post(notification: .pageScrolled, argument: accessibilityScrollText)
 
+    // ensure that scrolling related callbacks are still fired when performing scrolling via accessibility
+    if let visibleDayRange {
+      didEndDecelerating?(visibleDayRange)
+      didEndDragging?(visibleDayRange, false)
+      didScroll?(visibleDayRange, true)
```

Comment:
isn't this order flipped? I would expect this order, and these parameters:

```suggestion
      didScroll?(visibleDayRange, false)
      didEndDragging?(visibleDayRange, true)
      didEndDecelerating?(visibleDayRange)      
```

Reply:
Yup, they are in the exact opposite order they should be in 🤦 . Thanks for catching that!

For the parameters, I thought we might want to set didScroll / isDragging to true since the action was performed by the user via voiceover? But maybe I'm misunderstanding how that parameter should be used.

For `didEndDragging`, I thought we would want to set `willDecelerate` to false since after this is fired there will be no further scrolling/deceleration?

Reply:
> For the parameters, I thought we might want to set didScroll / isDragging to true since the action was performed by the user via voiceover? But maybe I'm misunderstanding how that parameter should be used.

I think that logic makes sense actually, let's leave that one as true

> For didEndDragging, I thought we would want to set willDecelerate to false since after this is fired there will be no further scrolling/deceleration?

This one is a bit tricky - the accessibility scroll has an animation / what appears to be some deceleration period, right? I think we should just do whatever collection view does for these 2 calls.

Reply:
Confirmed collection view behavior - it was exactly as you suggested. Addressed via fixup.